pkg/web/assets/css/dashboard.css

```diff
@@ -12,6 +12,18 @@
   white-space: nowrap;
   overflow: hidden;
   margin-left: -2px; /* Fix for kerning issue */
+  display: inline;
```

Comment:
I'm no UI or CSS professional so any adjustments here are welcomed :)

Reply:
I will probably get someone else to help with a UI pass at this before we release so that we can clean it up.  I'm not a huge fan of the overview page style choices.